packages/opentelemetry-instrumentation-openai/opentelemetry/instrumentation/openai/v1/responses_wrappers.py

```
@@ -416,6 +461,12 @@
                     "model", existing_data.get("request_model", "")
                 ),
                 response_model=existing_data.get("response_model", ""),
+                # Reasoning attributes
+                request_reasoning_summary=kwargs.get("reasoning", {})\
+                            .get("summary", existing_data.get("request_reasoning_summary")),
```

Flake8 (7.2.0)

[error] continuation line unaligned for hanging indent

(E131)